data/homepage.yml

```diff
@@ -1,63 +1,73 @@
-  comment : "InVision is a window into everything that's being designed at Twitter. It gets all of our best work in one place."
-  clientImage : "images/testimonial/feature-testimonial-thumb.jpg"
-  clientName : "Jonathon Andrew , Themefisher"
+  image : "images/boxology_overview_nobar.webp"
+  imageAlt : "images/boxology_overview_nobar.png"
+  title : "A community-driven, multi-use quantum chemistry datasets that the CMS community can access free of charge"
+  content : 'The core of this project sets out to answer the fundamental question of "How do we compile, aggregate, query, and share quantum chemistry data to accelerate the understanding of new method performance, fitting of novel force fields, and supporting the incredible data needs of machine learning for computational molecular science?"'
+  comment : "[Do you want to be quoted on the website?] Sure!"  #LNN Note: Make a real quote of DGAS
+  clientImage : "images/people/dgashead.jpg"
+  clientName : "Daniel G. A. Smith, Head QCA Developer, MolSSI Software Scientist"
 
 # feature two
 featureTwo:
   enable : true
-  image : "images/feature/feature-new-02.webp"
-  imageAlt : "images/feature/feature-new-02.jpg"
-  title : "Increase your productivity with Small Apps"
-  content : "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
-  comment : "InVision is a window into everything that's being designed at Twitter. It gets all of our best work in one place."
-  clientImage : "images/testimonial/feature-testimonial-thumb.jpg"
-  clientName : "Jonathon Andrew , Themefisher"
+  image : "images/molssi-logo.webp"
+  imageAlt : "images/molssi-logo.png"
+  title : "A product of the Molecular Sciences Software Institute"
+  content : "The QC Archive project is developed by MolSSI and provides support to the broader Computational Molecular Sciences communities. It brings expertise in Big Data management, software development and deployment expertise, and infrastructure control that individual research groups may not have on their own. "
+  comment : "Science should not be gated behind a learning curve of hardware implementation, software distribution, cluster management, and parallel task deployment."
+  clientImage : "images/people/nadenhead.jpg"
+  clientName : "Levi Naden, QCA Developer, MolSSI Software Scientist"
 
 # service
 service:
   enable : true
-  title : "An Interface For Lifestyle"
-  description : "Small Apps makes it easy to stay on top of your Life Style. No late tasks. No gimmicks."
-  image : "images/feature/iphone-ipad.webp"
-  imageAlt : "images/feature/iphone-ipad.jpg"
+  title : "QC Archive Ecosystem"
+  description : "The QC Archive Ecosystem provides access at every step of the compute lifecycle. From simple result lookup to computation of proprietary basis sets on your hardware."
```

Comment:
better standardize this now. I vote for "QCArchive" over "QC Archive"

Reply:
My vote is on "QC Archive" with the space. The project logo is QC and I would distinguish between QCA / QCF / QCEn / QCEl (QCL?) / QCS as the short hands (im making up most of those) when using them like that, and QCArchive to talk about the explicit code or when spaces are not viable. Its still a proper noun so its capitalized when written.

Why do you prefer the no-space version?